cmd/runtimetest/main.go

```diff
@@ -178,7 +182,7 @@ func validateCapabilities(spec *rspec.Spec) error {
 	expectedCaps3 := make(map[string]bool)
 	expectedCaps4 := make(map[string]bool)
 	expectedCaps5 := make(map[string]bool)
-	if spec.Process.Capabilities != nil {
+	if spec.Process != nil && spec.Process.Capabilities != nil {
```

Comment:
I'd rather move this guard to the top of this function, and have:

```go
if spec.Process == nil || spec.Process.Capabilities == nil {
    return nil
}
```

I don't see the point to initializing `processCaps`, etc. when we aren't going to use them.

Reply:
updated, thanks.